packages/atlas/src/hooks/useCreateMember.ts

```diff
@@ -199,20 +161,67 @@ export const useCreateMember = () => {
             break
         }
 
-        onError(SignUpSteps.CreateMember)
+        onError?.()
         return
       }
     },
-    [
-      addBlockAction,
-      createNewMember,
-      displaySnackbar,
-      emailAlreadyRegisteredMemberId,
-      handleLogin,
-      joystream,
-      refetchUserMemberships,
-      setAnonymousUserId,
-    ]
+    [addBlockAction, avatarMutation, displaySnackbar, faucetMutation]
+  )
+
+  const createNewOrionAccount = useCallback(
+    async ({ data, onError, onStart, onSuccess }: SignUpParams<AccountFormData>) => {
+      onStart?.()
+
+      await cryptoWaitReady()
+      const keypair = keyring.addFromMnemonic(data.mnemonic)
+
+      const address = keypair.address
+
+      try {
+        // const { data: memberShipData } = await refetchUserMemberships()
+        // const lastCreatedMembership = memberShipData.memberships[memberShipData.memberships.length - 1]
```

Comment:
to remove as well? I know you asked me about this and we had a condition here, but can't recall why we actually needed that, so let's just remove this